adaptivefiltering/dataset.py

```diff
@@ -38,6 +41,80 @@ def __init__(self, filename, warning_threshold=750000):
                 )
             )
 
+    def generate_geoTif(
```

Comment:
As this is a user interface, I have strong opinions about names. I would like to `save_mesh`, as it goes well with `show_mesh`.

Reply:
As the program is also capable of generating the mesh without permanently saving it to the hard drive, I would suggest `write_mesh` instead.